openweathermap-ios/openweathermap-ios/MapViewController.swift

```diff
@@ -22,6 +22,14 @@ final class MapViewController: UIViewController {
         locationManager.startUpdatingLocation()
     }
     
+    func addAnnotation(location: CLLocationCoordinate2D) {
```

Comment:
The `add` prefix gives the idea of incrementation. Consider using something like `update`.